addon/core/schema.ts

```diff
@@ -12,6 +12,44 @@ import { isElement } from '@lblod/ember-rdfa-editor/utils/_private/dom-helpers';
 
 // const logger = createLogger('core/schema');
 
+export const classicRdfaAttrSpec = {
+  vocab: { default: undefined },
+  typeof: { default: undefined },
+  prefix: { default: undefined },
+  property: { default: undefined },
+  rel: { default: undefined },
+  rev: { default: undefined },
+  href: { default: undefined },
+  about: { default: undefined },
+  resource: { default: undefined },
+  content: { default: undefined },
+  datatype: { default: undefined },
+  lang: { default: undefined },
+  xmlns: { default: undefined },
+  src: { default: undefined },
+  role: { default: undefined },
+  inlist: { default: undefined },
+  datetime: { default: undefined },
+};
+
+export function getClassicRdfaAttrs(
```

Comment:
It strikes me that each usage of this seems to be in an `if (rdfaAware) { getRdfaAttrs(node) } else { getClassicRdfaAttrs(node) }` type construction. Would it be cleaner to have `getRdfaAttrs` take `rdfaAware` and have the `if` in just one place?

Reply:
I updated both the `getRdfaAttrs` and `rdfaAttrSpec` functions to take in an `rdfaAware` parameter, so this should reduce the code duplication by quite a bit.